egret/models/acopf.py

```diff
@@ -27,6 +27,7 @@
 import networkx
 from pyomo.common.collections.orderedset import OrderedSet
 from pyomo.contrib.fbbt import interval
+from ._utils import get_unique_bus_pairs, get_out_of_service_gens, get_out_of_service_branches
```

Comment:
We're missing this file.

More critically, there a bunch of similar helper functions in this file: https://github.com/grid-parity-exchange/Egret/blob/main/egret/model_library/transmission/tx_utils.py.

I'm not sure those are in the right place either, but I don't think we should have these types of functions in multiple files.

Reply:
I think this is just a ""fat fingers" editing error so I am trying a quick fix.